applications/jupyter/workloads-without-iap.example.tfvars

```diff
@@ -35,8 +35,8 @@ gcp_service_account           = "jupyter-service-account"
 add_auth = false
 brand = ""
 support_email = ""
-default_backend_service = ""
-service_name = ""
+k8s_ingress_name = ""
+k8s_backend_config_name = ""
 
```

Comment:
NIT: have k8s_backend_service_name here to make everything consistent? Even though it won't need it.